lib/api/core/models/repositories/roleRepository.js

```diff
@@ -130,27 +130,20 @@ RoleRepository.prototype.loadRole = function roleRepositoryLoadRole (roleId) {
 /**
  *
  * @param {Array} controllers
- * @param {int} from
- * @param {int} size
  */
-RoleRepository.prototype.searchRole = function roleRepositorySearchRole (controllers, from, size) {
-  var query = {};
-
-  if (controllers && Array.isArray(controllers)) {
-    query.bool = {should: []};
-
-    controllers.forEach(controller => {
-      query.bool.should.push({exists: {field : 'controllers.' + controller}});
+RoleRepository.prototype.searchRole = function roleRepositorySearchRole (controllers) {
```

Comment:
Is it relevant to have more than 20-30 roles ? So, is it really necessary to implement pagination on it ?